docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -200,6 +200,12 @@ into::
     - key: Environment
       value: dev
 
+.. versionadded:: 2.8
+
+``dict2items`` accepts 2 keyword arguments, ``key_name`` and ``value_name`` that allow configuration of the names of the keys to use for the transformation::
+
+    {{ files | dict2items(key_name='file', value_name='path') }}
```

Comment:
show the 'before/after' of the data so people get a clear picture of what happens

Reply:
+1 - the example you posted in the PR description works well @rwanyoike